node/pkg/watchers/evm/ccq.go

```diff
@@ -732,22 +617,24 @@ type EthCallDataIntf interface {
 	CallDataList() []*query.EthCallData
 }
 
-func ccqBuildBatchFromCallData(req EthCallDataIntf, callBlockArg interface{}) ([]rpc.BatchElem, []EvmCallData) {
+// ccqBuildBatchFromCallData builds two slices. The first is the batch submitted to the RPC call. It contains one entry for each query plus one to query the block.
```

Comment:
feel like I'm missing something - this function name is different but the calls remain to `ccqBuildBatchFromCallData`